test/integration/http2_flood_integration_test.cc

```diff
@@ -1117,10 +1117,12 @@ TEST_P(Http2FloodMitigationTest, WindowUpdate) {
   const auto request = Http2Frame::makeRequest(request_stream_id, "host", "/");
   sendFrame(request);
 
-  // Since we do not send any DATA frames, only 4 sequential WINDOW_UPDATE frames should
-  // trigger flood protection.
+  // Per the `5 + 2 * (opened_streams_ +
+  // max_inbound_window_update_frames_per_data_frame_sent_ * outbound_data_frames_)`
+  // formula without sending any DATA frames, 1 + 5 + 2 * (1 + 0) = 8
```

Comment:
`1 + 5 + 2 * (1 + 0) = 8` shouldn't this be `5 + 2 * (1 + 0) = 7` with a note that we need to send more than that?

Reply:
I've made math a bit more explicit similar to the codec_impl_test.